packages/Webkul/Admin/src/Resources/lang/ar/app.php

```diff
@@ -1396,6 +1396,7 @@
             'create-success'      => 'تم إنشاء العنوان بنجاح',
             'update-success'      => 'تم تحديث العنوان بنجاح',
             'success-mass-delete' => 'حذف العناوين بنجاح',
+            'invalid-format'      => 'تنسيق الضريبة المضافة غير صحيح',
```

Comment:
Add your new key in alphabetical order based on the first letter of the key in all lang files.

Reply:
Done.